src/App.js

```diff
@@ -1,12 +1,130 @@
 import React from 'react';
 import './App.css';
 
-function App() {
-  return (
-    <div className="App">
-      <h1>Dynamic list of posts</h1>
-    </div>
-  );
+import { getData } from './api/data';
+import { PostList } from './components/PostList/PostList';
+
+function getPostsWithUsers(posts, users, comments) {
+  return posts.map(post => ({
+    ...post,
+    user: users.find(item => item.id === post.userId),
+    comments: comments.filter(comment => comment.postId === post.id),
+  }));
+}
+
+const urlUsers = 'https://jsonplaceholder.typicode.com/users';
```

Comment:
подобного рода константы пишутся следующим образом:
```suggestion
const USERS_API_URL = 'https://jsonplaceholder.typicode.com/users';
```